src/Bridge/Symfony/Bundle/Command/SwaggerCommand.php

```diff
@@ -63,7 +66,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $documentation = new Documentation($this->resourceNameCollectionFactory->create(), $this->apiTitle, $this->apiDescription, $this->apiVersion, $this->apiFormats);
         $data = $this->documentationNormalizer->normalize($documentation);
-        $content = json_encode($data, JSON_PRETTY_PRINT);
+        if ($input->getOption('yaml')) {
```

Comment:
`$input->getOption('yaml') ? Yaml::dump($data) : json_encode($data, JSON_PRETTY_PRINT`?

Reply:
If you insist. :+1: I try to avoid those.